src/websocket/datapoint.repository.ts

```diff
@@ -0,0 +1,19 @@
+import { PrismaService } from "../prisma.service";
+import { Injectable } from "@nestjs/common";
+
+@Injectable()
+export class DatapointRepository {
+  constructor(private prisma: PrismaService) {}
+
+  async createDatapoint(params: {
+    timestamp: Date;
+    co2: number;
+    humidity: number;
+    temperature: number;
+  }): Promise<void> {
+    const { timestamp, co2, humidity, temperature } = params;
+    this.prisma.datapoint.create({
+      data: { timestamp, co2, humidity, temperature },
+    });
+  }
```

Comment:
Rimelig sikker på det er best practice at return det objekt der bliver created. Derudover kan vi lade TypeScript infer return typen her.

```suggestion
  }) {
    const { timestamp, co2, humidity, temperature } = params;
    return this.prisma.datapoint.create({
      data: { timestamp, co2, humidity, temperature },
    });
  }
```